File: cf_java/prob_2057A.java
import java.util.Scanner;

public class prob_2057A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t-- > 0) {
            int n = sc.nextInt();
            int m = sc.nextInt();
            int minDim = Math.max(n, m);
            int result = 1 + minDim;
            System.out.println(result);
        }
        sc.close();
    }
}

File: cf_cpp/probs/prob_2057A.cpp
#include <bits/stdc++.h>
using namespace std;

void solve_prob_2057A() {
    int n, m;
    cin >> n >> m;
    int minDim = max(n, m);
    int result = 1 + minDim;
    cout << result << endl;
}

void prob_2057A() {
    int t = 1;
    cin >> t;
    while (t--) {
        solve_prob_2057A();
    }
}